src/main/java/org/apache/parquet/hadoop/BdecParquetWriter.java

```diff
@@ -35,6 +37,7 @@
 public class BdecParquetWriter implements AutoCloseable {
   private final InternalParquetRecordWriter<List<Object>> writer;
   private final CodecFactory codecFactory;
+  private long rowsWritten = 0;
```

Comment:
```
rowsWritten = 0;
```
also in close?

Reply:
Closing the writer doesn't clean up the written rows, so se should reset the counter. For example, the Parquet footer is only accessible after closing the writer.